src/test/ui/lint/lint-stability.rs

```diff
@@ -191,11 +191,11 @@ mod inheritance {
         stable_mod::unstable(); //~ ERROR use of unstable library feature
         stable_mod::stable();
 
-        unstable_mod::deprecated();
+        unstable_mod::deprecated(); //~ ERROR use of unstable library feature
         unstable_mod::unstable(); //~ ERROR use of unstable library feature
 
         let _ = Unstable::UnstableVariant; //~ ERROR use of unstable library feature
-        let _ = Unstable::StableVariant;
+        let _ = Unstable::StableVariant; //~ ERROR use of unstable library feature
```

Comment:
I was surprised these test cases were already present. I don't think we have any examples of either of these kinds of situations currently that we'll be breaking by changing this behavior, but I'm curious if there was any explicit justification for this behavior back when these test cases were added.

Reply:
Is this still true?  I don't see this stabilization in the PR.

Reply:
Ah, no, I removed the stabilization, we decided to break it because we don't think anyone is depending on it, pending crater run